components/Notification/Notification.tsx

```diff
@@ -0,0 +1,18 @@
+interface NotificationProps {
+  children: JSX.Element;
+  hasIcon?: boolean;
+}
+export const Notification = ({
+  children,
+  hasIcon = false,
+}: NotificationProps) => {
+  return (
+    <div
+      className={`flex justify-between bg-turquoise/[0.1] rounded-lg py-8 pr-[83px] ${
+        hasIcon ? " pl-[7px]" : " pl-[83px]"
```

Comment:
```suggestion
        hasIcon ? "justify-between pl-[7px]" : "justify-center"
```
Acá se podría remplazar el `padding` con `justify-content`. Además los espacios antes de los estilos no son necesarios, ya que hay un espacio antes del template literal